packages/core/src/express/decorators/express.ts

```diff
@@ -48,8 +48,8 @@ type Stage = 'before' | 'after';
  * @param middlewareFunction
  * @param stage
  */
-const middleware = (middlewareFunction, stage: Stage = 'before'): Function => {
-	return function(target: Record<string, any> | Function, methodName: string) {
+const middleware = (middlewareFunction, stage: Stage = 'before'): ClassDecorator & MethodDecorator => {
```

Comment:
shouldn't this be `ClassDecorator | MethodDecorator`?

Reply:
(For future reference...) According to [this StackOverflow comment](https://stackoverflow.com/questions/45306782/typescript-declaration-for-polymorphic-decorator#comment77857683_45306782) and [this similar code-change in Nest.js](https://github.com/nestjs/nest/commit/e8da23cd120e688ec97b5715184106a6c8061641#diff-2e475502839659c99ee212bbb1c78aaa8bdc15704e146433e56c31f9ac2d79c5R30) using `&` is the correct choice here.